package.json

```diff
@@ -39,7 +39,8 @@
     "test.karma": "karma start",
     "prepare": "husky install",
     "prerelease": "patch-package",
-    "release": "yarn prerelease && yarn workspaces foreach run release"
+    "release": "yarn prerelease && yarn workspaces foreach run release",
+    "jest": "jest"
```

Comment:
I added this simply to make it easy to run the jest that is installed in node_modules.
Example: `yarn run jest packages/rulesets/src/oas/functions/__tests/oasExample.test.ts`.